pandas/tests/frame/test_convert_to.py

```diff
@@ -175,7 +171,108 @@ def test_to_records_index_name(self):
     def test_to_records_with_unicode_index(self):
         # GH13172
         # unicode_literals conflict with to_records
-        result = DataFrame([{u'a': u'x', u'b': 'y'}]).set_index(u'a')\
+        result = DataFrame([{u'a': u'x', u'b': 'y'}]).set_index(u'a') \
             .to_records()
         expected = np.rec.array([('x', 'y')], dtype=[('a', 'O'), ('b', 'O')])
         tm.assert_almost_equal(result, expected)
+
+    def test_to_records_with_tz_gmt(self):
+        # GH13937
+        df = DataFrame({'datetime': date_range('2016-01-01',
+                                               periods=10,
+                                               freq='S',
+                                               tz='UTC')},
+                       index=date_range('2016-01-01',
+                                        periods=10,
+                                        freq='S',
+                                        tz='UTC'))
+
+        dfgmt = df.tz_convert("GMT")
```

Comment:
use ``result = ....``

can you construct a rec-array directly and use ``tm.assert_numpy_array_equal``